src/index.ts

```diff
@@ -15,10 +18,10 @@ mongoose
     })
     .then(() => {
         console.log('Successfully connected to MongoDB database');
-    
         const app = express();
 
         app.use('/api', apiRoutes);
+        app.use('/api/users', userRoutes);
```

Comment:
You can move this to `src/routes/api/index.ts` and just have `router.use('/users', userRoutes)` since it'll be under the router for `/api`